app.js

```diff
@@ -16,6 +16,9 @@ const INJECTED_HTML = `
     /* hide everything having the BE Feature banner */
     .be-indicator-container, .limited-be {display: none !important;}
 
+    /* hide Authentication method "Microsoft Active Directory" */
+    .app-react-components-BoxSelector-BoxSelectorItem-module__business {display: none !important;}
```

Comment:
I don't see a problem. But I must admit I do not know Portainer's HTML+CSS so well. I installed the software a couple of times and used it (before the whole BE stuff).
I would not recommend `nth-child` or something positional, because this can break easily. Also I noticed after creating the PR, that my rule not only eliminates the one box I show in the screenshot, but also a couple more further at the bottom, depending what authentication method you select.

Do you think the `BoxSelector-BoxSelectorItem` is auto-generated because of uppercase letters? Does look like a legitimate name (ok a stupid name because of mixing snake and camel cases :see_no_evil: ), but not special/auto-generated/about to change soon.

Reply:
Since portainer is open source, you can have a look at the source code to see where the class name comes from:

https://github.com/portainer/portainer/blob/6a51b6b41e9ecd79293a5ed8553f54d2c2286058/app/react/components/BoxSelector/BoxSelectorItem.module.css#L42-L49

https://github.com/portainer/portainer/blob/6a51b6b41e9ecd79293a5ed8553f54d2c2286058/app/react/components/BoxSelector/BoxSelectorItem.tsx#L46-L50

The class name is indeed transformed (rather than generated), so I assume that it won't change in near future.

If we want a bit more future-proof, I'd suggest matching all classes `[class*="BoxSelectorItem-module__business"]`, what do you think?